server/Mistakes.Journal.Api/Logic/Mistakes/Models/Mistake.cs

```diff
@@ -16,14 +15,25 @@ public class Mistake
         public ICollection<Tip> Tips { get; set; }
         public ICollection<MistakeLabel> MistakeLabels { get; set; }
         public bool IsSolved { get; set; }
+        public string Consequences { get; set; }
+        public string WhatCanIDoBetter { get; set; }
+        public string WhatDidILearn { get; set; }
+        public string CanIFixIt { get; set; }
+        public string OnlyResponsible { get; set; }
```

Comment:
I would create internal class `AdditionalQuestions` that contains such data.

Reply:
Remember, before that revert previous migration.

Reply:
Ok, but problems are possessed. We can talk about it